easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.4.1-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,208 @@
+    ('flatbuffers', '1.12.0'),
+    ('giflib', '5.2.1'),
+    ('hwloc', '1.11.12'),
+    ('ICU', '64.2'),
+    ('JsonCpp', '1.9.3'),
+    ('libjpeg-turbo', '2.0.3'),
+    ('LMDB', '0.9.24'),
+    ('NASM', '2.14.02'),
+    ('nsync', '1.24.0'),
+    ('SQLite', '3.29.0'),
+    ('PCRE', '8.43'),
+    ('protobuf-python', '3.10.0', versionsuffix),
+    ('flatbuffers-python', '1.12', versionsuffix),
+    ('typing-extensions', '3.7.4.3', versionsuffix),
+    ('libpng', '1.6.37'),
+    ('snappy', '1.1.7'),
+    ('zlib', '1.2.11'),
+]
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
+use_pip = True
+
+# Dependencies created and updated using findPythonDeps.sh:
+# https://gist.github.com/Flamefire/49426e502cd8983757bd01a08a10ae0d
+exts_list = [
+    ('Markdown', '3.3.3', {
+        'checksums': ['5d9f2b5ca24bc4c7a390d22323ca4bad200368612b5aaa7796babf971d2b2f18'],
+    }),
+    ('pyasn1-modules', '0.2.8', {
+        'checksums': ['905f84c712230b2c592c19470d3ca8d552de726050d1d1716282a1f6146be65e'],
+    }),
+    ('rsa', '4.7', {
+        'checksums': ['69805d6b69f56eb05b62daea3a7dbd7aa44324ad1306445e05da8060232d00f4'],
+    }),
+    ('cachetools', '4.2.0', {
+        'checksums': ['3796e1de094f0eaca982441c92ce96c68c89cced4cd97721ab297ea4b16db90e'],
+    }),
+    ('google-auth', '1.24.0', {
+        'modulename': 'google.auth',
+        'checksums': ['0b0e026b412a0ad096e753907559e4bdb180d9ba9f68dd9036164db4fdc4ad2e'],
+    }),
+    ('oauthlib', '3.1.0', {
+        'checksums': ['bee41cc35fcca6e988463cacc3bcb8a96224f470ca547e697b604cc697b2f889'],
+    }),
+    ('requests-oauthlib', '1.3.0', {
+        'checksums': ['b4261601a71fd721a8bd6d7aa1cc1d6a8a93b4a9f5e96626f8e4d91e8beeaa6a'],
+    }),
+    ('google-auth-oauthlib', '0.4.2', {
+        'checksums': ['65b65bc39ad8cab15039b35e5898455d3d66296d0584d96fe0e79d67d04c51d9'],
+    }),
+    ('Werkzeug', '1.0.1', {
+        'checksums': ['6c80b1e5ad3665290ea39320b91e1be1e0d5f60652b964a3070216de83d2e47c'],
+    }),
+    ('absl-py', '0.10.0', {
+        'modulename': 'absl',
+        'checksums': ['b20f504a7871a580be5268a18fbad48af4203df5d33dbc9272426cb806245a45'],
+    }),
+    ('astunparse', '1.6.3', {
+        'checksums': ['5ad93a8456f0d084c3456d059fd9a92cce667963232cbf763eac3bc5b7940872'],
+    }),
+    ('grpcio', '1.32.0', {
+        'modulename': 'grpc',
+        'checksums': ['01d3046fe980be25796d368f8fc5ff34b7cf5e1444f3789a017a7fe794465639'],
+    }),
+    ('tensorboard-plugin-wit', '1.8.0', {
+        'source_tmpl': 'tensorboard_plugin_wit-%(version)s-py3-none-any.whl',
+        'unpack_sources': False,
+        'checksums': ['2a80d1c551d741e99b2f197bb915d8a133e24adb8da1732b840041860f91183a'],
+    }),
+    ('tensorboard', version, {
+        'source_tmpl': 'tensorboard-%(version)s-py3-none-any.whl',
+        'unpack_sources': False,
+        'checksums': ['7b8c53c396069b618f6f276ec94fc45d17e3282d668979216e5d30be472115e4'],
+    }),
```

Comment:
What do you guys think about additional plugins here? I'm looking at tensorboard-plugin-profile and maybe tensorboard-plugin-neutron

Reply:
The first does indeed have crossed my mind already. It doesn't seem to have any dependencies so I'd add it. Not sure about the second one but could be useful. It needs `netron` though so a bit heavier.
Question then would be if user might often enough want to update those plugins or not

Are those TF version specific? If not then maybe it's better to tell users to pip install them into their home or so.

So I'm undecided but would agree in adding them

Reply:
We already had tensorboard-plugin-wit (doesn't seem like it would be required or anything)
I'm mostly interested in the profiler. The versioning and release history might indicate that it's version specific
https://pypi.org/project/tensorboard/#history
https://pypi.org/project/tensorboard-plugin-profile/#history
so I think this one belongs here.

Reply:
Agreed. Added it and put the 2 next to each other and below tensorboard which looks more logical